docs/experimental-features/host-risk-score.asciidoc

```diff
@@ -24,91 +24,248 @@ The following table shows how risk levels are applied to a host, based on the no
+
+In {kib}, run the following commands in Console to query the host risk score index: 
+
+[source,console]
+----------------------------------
+GET ml_host_risk_score_{{spaceID}}/_search
+GET ml_host_risk_score_latest_{{spaceID}}/_search
+----------------------------------
+
+If no data returns, you'll need to check if the source index (.`alerts-security.alerts-{{spaceID}}`) is generating alerts data:
+
+[source,console]
+----------------------------------
+GET _transform/<transform_id>/_stats?human=true
+----------------------------------
+
+Here's an example response: 
+
+[source,console]
+----------------------------------
+{
+  "count": 1,
+  "transforms": [
+    {
+      "id": "ml_hostriskscore_pivot_transform_default",
+      "state": "started",
+      "node": {
+        "id": "H1tlwfTyRkWls-C0sarmHw",
+        "name": "instance-0000000000",
+        "ephemeral_id": "SBqlp5ywRuuop2gtcdCljA",
+        "transport_address": "10.43.255.164:19635",
+        "attributes": {}
+      },
+      "stats": {
+        "pages_processed": 29,
+        "documents_processed": 11805,
+        "documents_indexed": 8,
+        "documents_deleted": 0,
+        "trigger_count": 9,
+        "index_time_in_ms": 52,
+        "index_total": 7,
+        "index_failures": 0,
+        "search_time_in_ms": 201,
+        "search_total": 29,
+        "search_failures": 0,
+        "processing_time_in_ms": 14,
+        "processing_total": 29,
+        "delete_time_in_ms": 0,
+        "exponential_avg_checkpoint_duration_ms": 59.02353261024906,
+        "exponential_avg_documents_indexed": 0.8762710605864747,
+        "exponential_avg_documents_processed": 1664.7724779548555
+      },
+      "checkpointing": {
+        "last": {
+          "checkpoint": 8,
+          "timestamp": "2022-10-17T14:49:50.315Z",
+          "timestamp_millis": 1666018190315,
+          "time_upper_bound": "2022-10-17T14:47:50.315Z",
+          "time_upper_bound_millis": 1666018070315
+        },
+        "operations_behind": 380,
+        "changes_last_detected_at_string": "2022-10-17T14:49:50.113Z",
+        "changes_last_detected_at": 1666018190113,
+        "last_search_time_string": "2022-10-17T14:49:50.113Z",
+        "last_search_time": 1666018190113
+      }
+    }
+  ]
+}
+----------------------------------
+
+Take note of the value from `time_upper_bound_millis` and enter it as a range query for the alerts index:
+
+[source,console]
+----------------------------------
+GET .alerts-security.alerts-default/_search
+{
+  "query": {
+    "range": {
+      "@timestamp": {
+        "lt": <time_upper_bound_millis>
+      }
+    }
+  }
+}
+----------------------------------
+
+Example: 
+
+[source,console]
+----------------------------------
+GET .alerts-security.alerts-default/_search
+{
+  "query": {
+    "range": {
+      "@timestamp": {
+        "lt": 1666018070315
+      }
+    }
+  }
+}
+----------------------------------
+
+If there's no response, install the user risk module again.
```

Comment:
If there's no response, please check the relevant [rules](https://www.elastic.co/guide/en/security/current/rules-ui-management.html) are running and alerts data are generated. Otherwise please **restart** the module and allow an hour for the data to be generate.